### PR TITLE
Feature/dvad 3253 node upgrade

### PR DESCRIPTION
**Pull request by JamesWoodwardHMPO**
This PR upgrades lev-report to run on Node 16.

https://collaboration.homeoffice.gov.uk/jira/browse/DVAD-3253

It also changes the default for POSTGRES_SSL from false to true, so it is consistent with lev-api.

### CODE REVIEW

**Review by aarongill1 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -3,11 +3,29 @@ Reports showing usage of LEV.
...
-npm i
+npm install --legacy-peer-deps
 npm test
 DB_USER=username DB_PASSWORD=password DB_HOST=hostname DB_PORT=port DB_DB=name npm start
 ```
 
+### Note
+The --legacy-peer-deps parameter is required by Node 16 as it has much stricter peer dependency checking than Node 12.
+
+## Local Developement & Testing
```
Comment (aarongill1) [inline README.md:14]: typo in development

File: README.md [hunk 2]
```diff
@@ -3,11 +3,29 @@ Reports showing usage of LEV.
...
+POSTGRES_DB=xxx POSTGRES_USER=xxx POSTGRES_PASSWORD=xxx POSTGRES_SSL=true NODE_TLS_REJECT_UNAUTHORIZED=0 npm start
+```
+#### Note
+- Insert appropriate values for the environment variables.
+- Start the postgres proxy for the environment
+- Set NODE_TLS_REJECT_UNAUTHORIZED=0 to allow self signed certificates during local testing
+
+### Test against a locally running database
+```
+docker compose up --build
```
Comment (aarongill1) [inline README.md:26]: docker-compose with the hyphen is required

**Review by aarongill1 — APPROVED** [review]
Just some minor comments but looks good and approved

File: README.md (hunk 2 above)
Comment (JamesWoodwardHMPO) [reply README.md:26]: Depends which version of Decker Desktop you have, docker-compose command has now been included in docker command.

File: README.md (hunk 1 above)
Comment (JamesWoodwardHMPO) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (aarongill1) [reply README.md:26]: You live and learn...

**Pull request merged**